packages/enzyme/src/ReactWrapper.js

```diff
@@ -886,6 +886,38 @@ class ReactWrapper {
     });
   }
 
+  /**
+   * Returns a new `ReactWrapper` around the node provided to the prop
+   *
+   * @param {String} propName
+   * @returns {ReactWrapper}
+   */
+  wrapProp(propName) {
+    const adapter = getAdapter(this[OPTIONS]);
+    if (typeof adapter.wrap !== 'function') {
+      throw new RangeError('your adapter does not support `wrap`. Try upgrading it!');
+    }
+
+    return this.single('wrapProp', (n) => {
+      if (n.nodeType === 'host') {
+        throw new TypeError('ReactWrapper::wrapProp() can only be called on custom components');
+      }
+      if (typeof propName !== 'string') {
+        throw new TypeError('ReactWrapper::wrapProp(): `propName` must be a string');
+      }
+      const props = this.props();
+      if (!has(props, propName)) {
+        throw new Error(`ReactWrapper::wrapProp(): no prop called "${propName}" found`);
+      }
+      const node = props[propName];
+      if (!adapter.isValidElement(node)) {
+        throw new TypeError(`ReactWrapper::wrapProp(): prop "${propName}" does not contain a valid element`);
+      }
+
+      return new ReactWrapper(node);
```

Comment:
what about the options? i'd expect `wrapProp` to take optional `options`.

Reply:
What kind of `options`? 

If you mean `props`, I didn't see how to use them, because you will have a mounted react element as a `prop`, and that's why you want to wrap it, else you use `renderProp`. In this case doesn't make sense to have `props` for the wrapped component.

Reply:
Not props, constructor options.

Reply:
Ah ok! I'll include those